packages/tokens/package.json

```diff
@@ -107,13 +112,18 @@
   },
   "scripts": {
     "build": "pnpm clean && node scripts/build.js",
-    "clean": "rimraf build",
+    "build:figma": "node scripts/figma/transform-tokens.js",
+    "fetch:figma": "pnpm run fetch:variables && pnpm run fetch:styles",
+    "fetch:styles": "figma-export use-config",
+    "fetch:variables": "node scripts/figma/fetch-variables.js && pnpm build:figma",
+    "clean": "rimraf dist",
     "lint:css": "stylelint \"dist/**/*.css\"",
     "lint:js": "eslint scripts dist/js",
     "lint:types": "tsc --noEmit",
     "lint": "pnpm run /^lint:.*/",
     "format.eslint": "pnpm lint:js --fix",
     "start": "pnpm build",
+    "_test": "pnpm build && node test/test-css-variables.js",
```

Comment:
What do you think about naming this "compare" or similar? I think "test" is a bit misleading here.